src/routes/verifyToken.ts

```diff
@@ -65,20 +67,13 @@ export async function verifyTokenRequest(req: VerifyTokenRequest, res: Response)
 async function checkAllGumroadProducts(licenseKey: string): Promise<boolean> {
     for (const link of config.gumroad.productPermalinks) {
         try {
-            const formData = new FormData();
-            formData.append("product_permalink", link);
-            formData.append("license_key", licenseKey);
-
-            const result = await axios.request({
-                url: "https://api.gumroad.com/v2/licenses/verify",
-                data: formData,
-                method: "POST",
-                headers: formData.getHeaders()
+            const result = await axios.post("https://api.gumroad.com/v2/licenses/verify", {
```

Comment:
This is still form data?

Reply:
https://help.gumroad.com/article/76-license-keys it can be done in parameters, doesn't need to be form data